### PR TITLE
More Secondary Languages 

### PR DESCRIPTION
**Pull request by MarsM0nd**
Adds secondary languages for Vox, Kidan, Tajaran, Diona, and IPC, all
having non-verbal components. This does not remove their current language.
Additional languages/better lore for the current languages would be apreciated.
The Tajaran language is copied from Bay, besides the change that it's both using verbal and non-verbal components, as a pure sign language would have quite some balance concern, and shouldn't be locked to a species if added at all.
#7536 Does also affect the added Siik'Maas, i will look into fixing it.
:cl:
add: Vox, Kidan, Tajaran, Diona, and IPC can now choose a species specific language besides their normal one.
/:cl:

### CODE REVIEW

**Comment by tigercat2000** [comment]
Why? This feels like it's just going to further reduce the actual usage of the languages, because now you can't even guarantee that two members of the same species will have their native language.

**Comment by scrubmcnoob** [comment]
Maybe add more languages on top of tradeband, gutter, and clownish instead?

**Comment by MarsM0nd** [comment]
I know it might have quite a low usage, but i figured it might be a nice addition to make the races a bit more alien, without affecting much of the balance. The languages are intended to be more of a dialect, so not every member of the species will be able to speak that language. The current languages are not touched, this just gives some characters another choice between Clownish, Gutter, and Tradeband. It also utilises the otherwise unused ``NONVERBAL`` flag, which makes the language basically unusable on comms, and adds the flavour of using signs sometimes.
In short, it should be non-balance affecting fluff that can't hurt much.

**Comment by Fox-McCloud** [comment]
Adding more primary langauges (these aren't secondary) is generally going to be frowned upon, for the reasons @tigercat2000 mentioned. Having one unique languages per species is more than enough.

**Comment by MarsM0nd** [comment]
They are choosen like secondary languages, just only chooseable by certain races. You can't be sure that the other vox you are talking to has the Vox-apgin selected as secondary language, but neither can you tell whether they have clownish, gutter, or tradeband. It would be some fluff you can expand upon, not really affecting balance, partly as the sub-group that would have that language can't just communicate on comms, would have to talk that everyone that plays the species does understand it. Tigercats concern was that "you can't even guarantee that two members of the same species will have their native language", but this way, they do keep their native language, just won't be able to guarantee they speak the dialect.
I know theese species specific language does have a weird mixture of primary and secondary language, but you keep the normal primary, and the rest is fluff that might add some RP possibilities.

**Comment by Allfd** [comment]
I don't mind the additional ones, however I would point out that Vox-pidgin is likely referring to a pidgin dialect, which is incorrect usage as arguably that is what their version of common is.  Just thought I would point that out as the words were changed around in theirs.

**Comment by Aurorablade** [comment]
I would actually like this is we somehow got around the issues fox and tiger brought up.

Though as i am understanding it, you have it as a secodnary pick like gutter/clownish/etc and they also know their native racial?

I mean i CAN accept that as long as the native racial remains.

**Comment by MarsM0nd** [comment]
Yes, it's adding the languages to the list of choosable languages for your secondary (or rather tertiary, if you count the species language as primary, and galactic common as secondary)  language, which is currently just gutter/clownish/tradeband, by going through the languages that exist, if the language is ``RACERESTRICTED``, it checks if your current species is in a list from that language, if it is, you can choose it. It's a list to allow multiple species to have the same, could have human + skrell language, coming from their first contact, i think lorewise it was only human + skrell in space for a while.
The language you get from your species is untouched.

**Comment by tigercat2000** [comment]
After discussion with the Heads of Staff and Maintainers, it was decided to close this PR. Adding more languages to these races would further pollute the interactions between their members due to even more potential language mismatches and, more importantly, the language prefixes.
